dapp_manager/dapp_starter.py

```diff
@@ -36,8 +36,15 @@ def start(self, timeout: float) -> None:
 
         success, stdout, stderr = self._check_succesful_startup(proc, timeout)
         if not success:
-            runner_stdout = self.storage.read_file("stdout")
-            runner_stderr = self.storage.read_file("stderr")
+            try:
+                runner_stdout = "".join(data for _, data in self.storage.iter_file_chunks("stdout"))
+            except FileNotFoundError:
+                runner_stdout = None
+
+            try:
+                runner_stderr = "".join(data for _, data in self.storage.iter_file_chunks("stderr"))
```

Comment:
Raw string manipulation here bothers me. Having additional method in `storage` that return complete content of a file would work better IMO. `self.storage.read_file("stdout")` was more cleaner.